### PR TITLE
BLUEBUTTON-984: Pull Data Servers from use before updating

### PR DESCRIPTION
**Pull request by karlmdavis**
This prevents the Data Servers from receiving queries (that they'd return errors for) while they're being updated. JBoss unfortunately doesn't support zero-downtime deploys or reconfigs, so this is necessary.

Note: this is a simplification of a similar branch that I was working on with Dave: I found a couple modules (`ec2_metadata_facts` and `elb_instance`) that work a lot easier and better.

https://jira.cms.gov/browse/BLUEBUTTON-984

### CODE REVIEW

**Comment by karlmdavis** [comment]
Note that a test deploy of this out in TEST worked -- the status is just confused because I declined to deploy to DPR and PROD.

https://builds.ls.r53.cmsfhir.systems/jenkins/job/bluebutton-ansible-playbooks-data/job/BLUEBUTTON-984-fix-404-errors-on-deployments-simplified/

File: Jenkinsfile [hunk 1]
```diff
@@ -130,6 +130,7 @@ stage('Deploy to TEST') {
 			}
 		}
 	} else {
+		echo 'Deployment skipped, likely due to being on a non-master branch.'
```
Comment (karlmdavis) [inline Jenkinsfile]: I'm cheating and slipping this unrelated change in, because not having this log message kept confusing me: why did the build not do anything, but marked itself successful? (I'm kinda' dumb like that.)

File: backend.yml [hunk 2]
```diff
@@ -83,5 +107,14 @@
         mode: 'u=rw,g=,o='
       become: true
 
+    - name: Add Data Server Back to Load Balancers
+      elb_instance:
+        region: "{{ aws_region }}"
+        ec2_elbs: "{{ item }}"
+        instance_id: "{{ ansible_ec2_instance_id }}"
+        state: absent
```
Comment (jzulim) [inline backend.yml]: If adding the instance back in, should this be set to 'present' ?

Comment (karlmdavis) [reply backend.yml]: :-O
